src/tests/tests_runtime.rs

```diff
@@ -139,22 +142,29 @@ fn test_builtins() {
         "/wasm/use_builtins.wasm"
```

Comment:
I think in the test, that the function process.time can be enabled

Reply:
I don't understand what you mean by that

Reply:
Continue discussion in #218